app/services/IdentityService.scala

```diff
@@ -72,4 +85,10 @@ class IdentityService(wsClient: WSClient, config: Config)(implicit ec: Execution
           error("Error accessing Identity API", err)
           Autofill.empty
       }
+
+  def updateMarketingOptIn(email: String, marketingOptIn: Boolean)(implicit tags: LoggingTags): Future[Boolean] = {
```

Comment:
you can delete `updateMarketingPreferences` from this class

Reply:
I can't delete this as this one as it is called from the Identity Controller, which is somewhere we want to call this from

Reply:
done